src/pages/en/guides/cms/keystonejs.mdx

```diff
@@ -0,0 +1,9 @@
+---
+title: KeystoneJS & Astro
+description: Add content to your Astro project using KeystoneJS as a CMS
+layout: ~/layouts/CMSLayout.astro
+stub: true
+service: KeystoneJS
+---
+
+[KeystoneJS](https://keystonejs.com/) helps you build faster and scale further than any other CMS or App Framework. Just describe your schema, and get a powerful GraphQL API & beautiful Management UI for content and data.
```

Comment:
```suggestion
[KeystoneJS](https://keystonejs.com/) helps you build faster and scale further than any other CMS or App Framework. Just describe your schema, and get a powerful GraphQL API & beautiful Management UI for content and data.
```
Let's not use their wording *exactly* here.  :sweat_smile: This isn't meant to be a marketing page for each, but more neutral. Can you try to redescribe like some of the others, just saying what they ARE and the tech they use, without making any particular claims about it?  :smile: